### DISCUSSION TITLE
Set Active Bar in Charts

### DISCUSSION THREAD

**Discussion by planecore** [mantinedev/mantine#6305] [Feature requests]
Hi! I'd love to get the ability to set the active bar in @mantine/charts, specifically the `BarChart`.

My use-case is letting the user click on a bar to show additional data in the website sidebar. I'd like the clicked-on bar to be in a different color so the user will have greater context.

**Comment by adaminspaceship** [comment]
I have the exact same use case! Would like this too...

**Comment by planecore** [comment]
I can add it myself if the maintainers will approve it, waiting for their response.

**Comment by rtivital** [comment]
Can you provide an example (screenshot of video) of what you mean by active bar chart?

**Comment by ktkennychow** [comment]
I think this one here:
https://codesandbox.io/p/sandbox/bar-chart-with-customized-event-mmwz85?file=%2Fsrc%2Findex.tsx
https://recharts.org/en-US/examples/BarChartWithCustomizedEvent
I wonder if this is already possible with the current version?

**Reply by ktkennychow** [reply]
I have a use case where this is needed in a stacked bar chart, each cell in a stacked bar should be clickable and be extracted data from. Would love to know if someone has done the same thing or should I use the rechart library for this.

**Comment by planecore** [comment]
Hi, how can we go forward with this?

**Reply by alopez0408** [reply]
any news on this?
